### PR TITLE
Lime 775

### PR DESCRIPTION
**Pull request by MathumithaVe**
Take remaining Welsh DL tests from API repo and move to DL FE repo
[P4-(https://govukverify.atlassian.net/browse/LIME-775)]
 PR Title` --> LIME-755

Moved remaining Driving Licence test to front end

### CODE REVIEW

File: test/browser/features/Welsh_DVA_Driving_Licence.feature [hunk 1]
```diff
@@ -62,6 +62,14 @@ Feature: DVA Driving licence CRI Error Validations
     Given I can see the postcode field titled Cod post
     Then I can see postcode sentence as Rhowch y cod post yn y cyfeiriad yn adran 8 o'ch trwydded
 
+  @mock-api:dva-ConsentSection @validation-regression @build @staging@Language-regression
```
Comment (gds-jbentham) [inline test/browser/features/Welsh_DVA_Driving_Licence.feature:65]: @MathumithaVe The welsh tests don't need the build, staging or validation regressions tags

File: test/browser/pages/DVADetailsEntryPage.js [hunk 2]
```diff
@@ -136,6 +136,17 @@ exports.DVADetailsEntryPage = class PlaywrightDevPage {
...
+
+    this.consentSectionSentenceOneDVA = this.page.locator(
+      'xpath=//*[@id="consentDVACheckbox-hint"]/p[1]'
+    );
+
+    this.consentSectionSentenceTwoDVA = this.page.locator(
+      'xpath=//*[@id="consentDVACheckbox-hint"]/p[2]'
+    );
+    this.privacyPolicyDVALink = this.page.locator("a", {
+      hasText: "hysbysiad preifatrwydd DVA (agor mewn tab newydd)",
```
Comment (gds-jbentham) [inline test/browser/pages/DVADetailsEntryPage.js:148]: Can we do this with xpath or an id we want to be able to reuse this outside of the welsh check

File: test/browser/pages/DVADetailsEntryPage.js [hunk 3]
```diff
@@ -391,4 +402,36 @@ exports.DVADetailsEntryPage = class PlaywrightDevPage {
...
+    expect(await this.privacyPolicyDVALink.innerText()).to.equal(
+      consentPrivacyLink
+    );
+  }
+
+  async assertDVARetryErrorMessage(retryMessageHeading) {
+    await this.page.waitForLoadState("domcontentloaded");
+    expect(await this.isCurrentPage()).to.be.true;
+    expect(
+      await this.drivingLicenceDVARetryMessageHeading.innerText()
```
Comment (gds-jbentham) [inline test/browser/pages/DVADetailsEntryPage.js:434]: Has this been declared drivingLicenceDVARetryMessageHeading?

File: test/browser/pages/DrivingLicencePage.js [hunk 4]
```diff
@@ -233,6 +233,38 @@ exports.DrivingLicencePage = class PlaywrightDevPage {
...
+    this.consentSectionSentenceOne = this.page.locator(
+      'xpath=//*[@id="consentCheckbox-hint"]/p[1]'
+    );
+
+    this.consentSectionSentenceTwo = this.page.locator(
+      'xpath=//*[@id="consentCheckbox-hint"]/p[2]'
+    );
+
+    this.oneLoginLink = this.page.locator("a", {
+      hasText: "hysbysiad preifatrwydd GOV.UK One Login (agor mewn tab newydd)",
```
Comment (gds-jbentham) [inline test/browser/pages/DrivingLicencePage.js:250]: Can we swap out this welsh for ids

File: test/browser/pages/DVADetailsEntryPage.js (hunk 3 above)
Comment (MathumithaVe) [reply test/browser/pages/DVADetailsEntryPage.js:434]: removed this method which is not used.

File: test/browser/features/WelshDrivingLicence.feature [hunk 5]
```diff
@@ -366,3 +373,33 @@ Feature: DVLA Driving licence CRI Error Validations
...
+    When User clicks on continue
+    And I can see the DVLA consent error on the checkbox as Mae'n rhaid i chi roi eich caniatâd i barhau
+    Then I can see the DVLA consent error summary as Mae'n rhaid i chi roi eich caniatâd i barhau
+    And I check the page Title Gwall: Rhowch eich manylion yn union fel maent yn ymddangos ar eich trwydded yrru – Profi pwy ydych chi – GOV.UK
+    Examples:
+      |DrivingLicenceSubject             |
+      |DrivingLicenceSubjectHappyPeter   |
+
+    ##### Retry message #####
+  @mock-api:dl-failed @build @staging @language-regression
```
Comment (gds-jbentham) [inline test/browser/features/WelshDrivingLicence.feature:391]: This still need build and staging tags removed

File: test/browser/features/Welsh_DVA_Driving_Licence.feature (hunk 1 above)
Comment (gds-jbentham) [inline test/browser/features/Welsh_DVA_Driving_Licence.feature:65]: This still needs build staging and validation tags removed

File: test/browser/features/Welsh_DVA_Driving_Licence.feature [hunk 6]
```diff
@@ -302,3 +310,28 @@ Feature: DVA Driving licence CRI Error Validations
...
+    And DVA consent checkbox is unselected
+    When User clicks on continue
+    Then I can see the DVA consent error summary as Mae'n rhaid i chi roi eich caniatâd i barhau
+    And I can see the DVA consent error on the checkbox as Mae'n rhaid i chi roi eich caniatâd i barhau
+    And I check the page Title Gwall: Rhowch eich manylion yn union fel maent yn ymddangos ar eich trwydded yrru – Profi pwy ydych chi – GOV.UK
+    Examples:
+      |DVADrivingLicenceSubject             |
+      |DrivingLicenceSubjectHappyBilly|
+
+  @mock-api:dl-failed @build @staging @language-regression
```
Comment (gds-jbentham) [inline test/browser/features/Welsh_DVA_Driving_Licence.feature:326]: This still need build and staging tags removed

**Review by ElliotMurphyGDS — DISMISSED** [review]

**Review by smsgds — APPROVED** [review]

**Pull request merged**